src/lib.rs

```diff
@@ -324,13 +329,13 @@ pub struct SectionRegex {
 // TODO: Document these?
 pub fn init_section_regex() -> SectionRegex {
     SectionRegex {
-        bill_section: Regex::new(r"^(?i)section(?-i)\s*(\d*)\s*\.").unwrap(),
+        bill_section: Regex::new(r"^\s*(?i)section(?-i)\s*(\d*)\s*\.").unwrap(),
         amended: Regex::new(r"amended").unwrap(),
         striking: Regex::new(r"striking").unwrap(),
         inserting: Regex::new(r"inserting").unwrap(),
         repealed: Regex::new(r"repealed").unwrap(),
-        law_chapter: Regex::new(r"^(?i)section(?-i).*?[cC]hapter\s*(\d*\w*)").unwrap(),
-        law_section: Regex::new(r"^(?i)section(?-i).*?([sS]ection[s]?)\s*(\d*\w*)").unwrap(),
+        law_chapter: Regex::new(r"^\s*(?i)section(?-i).*?[cC]hapter\s*(\d*\w*)").unwrap(),
+        law_section: Regex::new(r"^\s*(?i)section(?-i).*?([sS]ection[s]?)\s*(\d*\w*)").unwrap(),
```

Comment:
Probably should have left this for the regex PR, but this catches leading spaces before the section indicator

Reply:
Makes sense to do here.